.github/workflows/package-publish.yml

```
@@ -2,24 +1,88 @@
+name:  publish-npm-packages
 
 on: workflow_dispatch
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)